### PR TITLE
Nep 40 categories

### PR DESCRIPTION
**Pull request by shrijan-siris**
- getting folowing errror 
 https://github.com/facebook/react-native/issues/16332
- resolved but can't rendered the component

### CODE REVIEW

File: src/home/index.js [hunk 1]
```diff
@@ -13,56 +13,56 @@ import SplashScreen from './components/splash-screen'
...
-      velocityThreshold: 0.3,
-      directionalOffsetThreshold: 80
-    }
-    return (
-      <QueryRenderer
-        environment={environment}
-        variables={{ isUpdated: this.state.isUpdated }}
-        query={graphql`
+	render () {
+		// const config = {
```
Comment (syuraj) [inline src/home/index.js]: @shrijan-siris we should remove unused code

**Comment by syuraj** [comment]
@shrijan-siris this pr seems to have merge conflicts

**Comment by syuraj** [comment]
image
current state

**Comment by syuraj** [comment]
image
current state

File: src/home/index.js (hunk 1 above)
Comment (shrijan-siris) [reply src/home/index.js]: done

**Pull request merged**